### DISCUSSION TITLE
GDPR?

### DISCUSSION THREAD

**Discussion by Hi-there-how-are-u** [sws2apps/organized-app#2528] [Q&A] [answered]
Hi, just a question whether Organized is compliant with the GDPR we have here in the EU?
Cannot find anything about it and we are not allowed to use if it's not compliant with GDPR.

**Comment by rhahao** [comment, marked as answer]
Hello @Hi-there-how-are-u,
Yes, we are doing our best to make Organized as GPDR-compliant. We already have a link pointing to the [Privacy Policy](https://sws2apps.com/privacy) document when you register a new account. You may want to check this [Security and privacy](https://guide.organized-app.com/faq/security) guide as well.
But we will add some minor adjustment to the app to make this clearer. We hope this answers your question.

**Reply by fschein** [reply]
Hi! And what about LGPD from Brazil? Thank you.

**Reply by ux-git** [reply]
@fschein Hi! LGPD and GDPR share many core principles, including transparency, security, user consent, data minimization, and accountability. Since our app is GDPR-compliant, it's likely well-aligned with LGPD standards as well.

To learn more about how the app handles data, check out our [privacy policy](https://sws2apps.com/privacy) as mentioned. If needed, consider consulting a local legal expert familiar with Brazilian data protection laws to ensure the app meets the specific requirements. If you find any specific areas that should be improved, please let us know — we’re always eager to make improvements!
